app/models/compute_resources/foreman/model/vmware.rb

```diff
@@ -858,7 +861,7 @@ def cachekey_with_cluster(key, cluster_id = nil)
     # @param firmware [String] The firmware type.
     # @return [Hash] A hash with secure boot settings if applicable.
     def generate_secure_boot_settings(firmware)
-      firmware == 'uefi_secure_boot' ? { "secure_boot" => true } : {}
+      firmware == 'uefi_secure_boot' ? { secure_boot: true } : {}
```

Comment:
What's the reasoning for this change?

Reply:
The reason for the change is this line:
https://github.com/theforeman/foreman/pull/10524/files#diff-be4ac4f95deb530eb0fbcc834a4940573ba45b8d5923433305ac3aa1e0703328R596

I updated it so I can use the `:secure_boot` symbol key consistently with the rest of the opts hash, instead of mixing it with a string key ("secure_boot" => args["secure_boot"]).